index.html

```diff
@@ -0,0 +1,874 @@
+        </section>
+        <section class="border-top">
+          <img src="img/bootstrap.png" alt="Image Debate" class="float-left mr-4">
+          <h5 class="font-weight-bold"><a href="#">CSS is easier with bootstrap!</a></h5>
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        </section>
+      </article>
+    <!-- Culture & travel section -->
+      <article class="culture-travel bg-white">
+        <h5 class="text-muted text-uppercase">Culture & Travel</h5>
+        <div class="d-flex flex-column flex-md-row">
+          <section class="travel-container">
+            <figure class="img-container m-auto">
+          <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+          <figcaption class="category-image pl-2 pr-2 small">
+            Travel
+          </figcaption>
+            </figure>
+            <h5 class="font-weight-bold"><a href="#">Yes! It's the same winter landscape</a></h5>
+            <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          </section>
+          <section class="travel-container">
+            <figure class="img-container m-auto">
+              <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+              <figcaption class="category-image pl-2 pr-2 small">
+              Travel
+              </figcaption>
+            </figure>
+            <h5 class="font-weight-bold"><a href="#">Yes! It's the same winter landscape</a></h5>
+            <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          </section>
+          <section class="travel-container">
+            <figure class="img-container m-auto">
+              <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+              <figcaption class="category-image pl-2 pr-2 small">
+                Travel
+            </figcaption>
+            </figure>
+            <h5 class="font-weight-bold"><a href="#">Yes! It's the same winter landscape</a></h5>
+            <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          </section>
+        </div>
+      </article>
+    <!-- Latest News section -->
+      <article class="latest-news bg-white">
+        <h5 class="text-muted text-uppercase">Latest News</h5>
+          <section class="d-flex">
+            <img src="img/bootstrap.png" alt="Image Debate" class="float-left mr-4">
+            <h6 class="font-weight-bold clearfix"><a href="#">CSS is easier with bootstrap!</a></h6>
+          </section>
+          <section class="border-top d-flex">
```

Comment:
- [ ] The articles in this section have a larger image when you are in the medium breakpoint and below. Please adjust them.